app/models/form1095_b.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+class Form1095B < ApplicationRecord
+  has_kms_key
+  encrypts :form_data, key: :kms_key, **lockbox_options
+
+  # validations
+  validates :veteran_icn, :tax_year,  presence: true
+  validates :veteran_icn, uniqueness: { scope: :tax_year }
+  validate :proper_form_data_schema
+
+  # scopes
+  scope :get_available_forms, lambda { |icn|
+    where(veteran_icn: icn).order(tax_year: :desc).pluck(:tax_year, :updated_at)
+  }
+
+  # methods
+  def get_pdf
```

Comment:
`get` is redundant here.
```suggestion
  def pdf
```